classify-cluster-priority/README.md

```diff
@@ -0,0 +1,85 @@
+
+### Well-defined label values
+
+There are three well-defined label values to be used.
+
+- `giantswarm.io/service-priority: production`: A cluster labelled this way serves production traffic (i. e. end-user facing). Clusters with this label value are considered the most important ones, relative to other clusters not carrying this label.
+
+- `giantswarm.io/service-priority: staging`: Clusters carrying this label are used to simulate running on a production cluster, but are not serving production traffic. They are considered less important than ones carrying the above `production` label.
+
+- `giantswarm.io/service-priority: development`: A cluster with this label is used by developers to develop and test applications. It is considered less important than a cluster labelled as `production` or `staging`.
+
+### Absence of the label
+
+The absence of the label SHOULD be interpreted as if the label was present with the value `production`.
+
+### Using undefined values
+
+If the label is used with a value different from the well-defined ones, the label SHOULD be interpreted as if the value was `development`.
+
+## Additional considerations and explanations
+
+- Understanding of this system by Giant Swarm staff is crucial. We have to make sure that everybody in operations know about the system.
+
+- Individual customers may or may not use the system. It is not required that all customers adopt it for it to become usable.
+
+- Customers that already have a classification scheme implemented should be encouraged to also implement the system proposed here. Several systems can co-exist side by side, however in that case Giant Swarm staff will be instructed to consider the system described here as the authoritative one.
+
+- The label, by default, has no effect on monitoring, alerting, support, usage cost, routing priority, workload quality of service (QoS) etc. of the cluster. However it MAY serve as an input for such automation in the future.
+
+- The `kubectl gs template cluster` command SHOULD be extended to include the proposed label, with the default value being `production` and the other well-defined labels to be set via flags. This will help give the system more visibility and increase adoption.
+
+- While it is possible to set labels on clusters via the Management API or in a cluster manifests, it is also recommendable to extend the web UI to set the proposed label and choose a well-defined value on cluster creation.
+
+- While it is already possible to set, modify, and delete cluster labels via the web UI, it is also recommended to make it simpler for end users to specify the proposed label with a well-defined value for a given cluster.
+
```

Comment:
I'd also like to see `kubectl gs get clustets` output expanded to highlight this label if present.

Reply:
Ah! You have it below! :D Nice!

Reply:
Right. Will resolve the comment.